api/v1beta1/machinedeployment_types.go

```diff
@@ -57,6 +57,10 @@ const (
 	// MachineDeploymentUniqueLabel is the label applied to Machines
 	// in a MachineDeployment containing the hash of the template.
 	MachineDeploymentUniqueLabel = "machine-template-hash"
```

Comment:
There is code in rollout_rollbacker.go which drops this label when doing a rollback. Do we need the same for the new label?